ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -385,6 +385,37 @@ if test "${ret}" -eq 0 ; then
   touch  "${SHARED_DIR}/success"
   # Save console URL in `console.url` file so that ci-chat-bot could report success
   echo "https://$(env KUBECONFIG=${dir}/auth/kubeconfig oc -n openshift-console get routes console -o=jsonpath='{.spec.host}')" > "${SHARED_DIR}/console.url"
+
+  if [ ! -f /tmp/jq ]; then
+    curl -L https://stedolan.github.io/jq/download/linux64/jq -o /tmp/jq && chmod +x /tmp/jq
+  fi
+  #save cluster data
+  touch  "${ARTIFACT_DIR}/cluster-data"
+  # platform
+  echo "Platform: $(env KUBECONFIG=${dir}/auth/kubeconfig oc get infrastructure/cluster -o json|/tmp/jq '.status.platform')/n" >> "${ARTIFACT_DIR}/cluster-data"
+  # topology
+  TOPOLOGY="$(env KUBECONFIG=${dir}/auth/kubeconfig oc get node -o json|/tmp/jq '.items[0]|.metadata.annotations'|grep machineconfiguration.openshift.io/controlPlaneTopology)"
+  if [[ "$TOPOLOGY" =~ .*"HighlyAvailable".*  ]]; then
+    echo "Topology: ha/n" >> "${ARTIFACT_DIR}/cluster-data"
```

Comment:
What is the /n in these, and can we make it more specific in the json file?

Reply:
Are they meant to be newlines? \n

But they also may not be necessary when doing >>

Reply:
/n were experimental from earlier changes. They are fixed in new ones while making it a json file from a normal txt file